avro-fastserde/src/codegen/java/com/linkedin/avro/fastserde/generated/deserialization/AVRO_1_7/StringableRecord_SpecificDeserializer_6174384286732341990_6174384286732341990.java

```diff
@@ -107,6 +107,13 @@ public StringableRecord_SpecificDeserializer_1822942669987229249_182294266998722
         StringableRecord.put(6, urlMap0);
         StringableRecord.put(7, deserializeStringableSubRecord0(StringableRecord.get(7), (decoder)));
         StringableRecord.put(8, deserializeAnotherSubRecord0(StringableRecord.get(8), (decoder)));
+        int unionIndex0 = (decoder.readIndex());
+        if (unionIndex0 == 0) {
+            decoder.readNull();
+        }
+        if (unionIndex0 == 1) {
```

Comment:
Can this be an else if, since it should never be useful to evaluate it if the previous condition was true?

Reply:
Fixed.